src/main/java/com/minecolonies/coremod/tileentities/TileEntityColonyBuilding.java

```diff
@@ -287,7 +287,7 @@ public boolean isUsableByPlayer(final EntityPlayer player)
     public boolean hasAccessPermission(final EntityPlayer player)
     {
         //TODO This is called every tick the GUI is open. Is that bad?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)